Userland/Applications/Browser/BookmarksBarWidget.cpp

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright (c) 2020, Emanuel Sprung <emanuel.sprung@gmail.com>
  * Copyright (c) 2022, networkException <networkexception@serenityos.org>
+ * Copyright (c) 2023, the SerenityOS developers.
```

Comment:
"the SerenityOS developers" is a fallback for new files when people don't want to add their name to copyright. Adding it to an existing copyright is kinda pointless.

Reply:
Was not aware it was only for new files, thanks for the heads up.